web_responsive_company_color/models/res_company.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    def _get_scss_template(self):
+        return (
+            super()._get_scss_template()
+            + """
+             .o_menu_apps .dropdown-menu {
+                background: url('/web_responsive/static/img/home-menu-bg-overlay.svg'),
+                  linear-gradient(
+                    to bottom,
+                    %(color_navbar_bg)s,
+                    desaturate(lighten(%(color_navbar_bg)s, 20%%), 15)
+                  );
+             }
+            """
+        )
+      
+    def _get_scss_template_uninstall(self):
+        return super()._get_scss_template()
```

Comment:
**question**: What does this method do?.

Reply:
This method tries to remove web elements introduced by the module for customization during uninstall, so that when this module is uninstalled customization related to its web elements are removed.

During uninstall, web_company_color module tries to return self._get_scss_template() % self._scss_get_sanitized_values() using _scss_generate_content() method.

Whenever, self._get_scss_template() is called, it provides the entire scss_template including web element which has be removed for the module for which uninstalled is performed.

In order to avoid including web elements of module which are about to uninstall, calling a scss_template_uninstall method in the module which tries uninstall. As only parent of scss_template is called in this method, I am able to remove customized web elements of module for which uninstall is performed.

If not, though module is uninstalled, customization of web element introduced by the uninstalled module remains.

Reply:
**question:** shouldn't this be something like:
```py
return super()._get_scss_template() % super()._scss_get_sanitized_values
```